egs/fisher_english/s5/local/semisup/chain/tuning/run_tdnn_100k_semisupervised_1a.sh

```diff
@@ -0,0 +1,429 @@
+#!/bin/bash
+
+# Copyright 2017  Vimal Manohar
+# Apache 2.0
+
+# This script is semi-supervised recipe with 100 hours of supervised data
+# and 250 hours unsupervised data with naive splitting.
+# Based on "Semi-Supervised Training of Acoustic Models using Lattice-Free MMI",
+# Vimal Manohar, Hossein Hadian, Daniel Povey, Sanjeev Khudanpur, ICASSP 2018
+# http://www.danielpovey.com/files/2018_icassp_semisupervised_mmi.pdf
+# local/semisup/run_100k.sh shows how to call this.
+
+# This version of script uses only supervised data for i-vector extractor
+# training as against using the combined data as in run_tdnn_50k_semisupervised.sh.
+# We use 3-gram LM trained on 100 hours of supervised data. We do not have
+# enough data to do 4-gram LM rescoring as in run_tdnn_50k_semisupervised.sh.
+
+# This script uses phone LM to model UNK.
+# This script uses the same tree as that for the seed model.
+# See the comments in the script about how to change these.
+
+# Unsupervised set: train_unsup100k_250k
+# unsup_frames_per_eg=150
+# Deriv weights: Lattice posterior of best path pdf
+# Unsupervised weight: 1.0
+# Weights for phone LM (supervised, unsupervised): 3,2
+# LM for decoding unsupervised data: 4gram
+# Supervision: Naive split lattices
```

Comment:
Is this accurate, that you are using naive splitting?  You seem to be also using it for the other example scripts; but the paper seems to say that "smart splitting" is generally better.  Can you clarify which options control the type of splitting?